project/dexter-monitor/public/ctrl/overviewCtrl.js

```diff
@@ -0,0 +1,151 @@
+            cellClass: 'grid-align',    headerTooltip: 'Group name'},
+        {field:'allDeveloperCount',         displayName:'Developers',               width: '16%',
+            cellClass: 'grid-align',    headerTooltip: 'Number of developers',
+            aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'targetDeveloperCount',      displayName:'Target developers',        width: '16%',
+            cellClass: 'grid-align',    headerTooltip: 'Number of developers who should install Dexter',
+            aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'installedDeveloperCount',   displayName:'Installed developers',     width: '16%',
+            cellClass: 'grid-align',    headerTooltip: 'Number of developers who installed Dexter',
+            aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'installationRate',          displayName:'Installation rate (%)',    width: '16%',
+            cellClass: 'grid-align',    headerTooltip: 'Installation rate',
+            aggregationType: () => $scope.rate, footerCellClass: 'grid-align'},
+        {field:'nonTargetDeveloperCount',   displayName:'Non-target developers',    width: '19%',
+            cellClass: 'grid-align',    headerTooltip: 'Number of developers not applicable to install Dexter',
+            aggregationType: uiGridConstants.aggregationTypes.sum}
+    ];
+
+    const defectStatusColumnDefs = [
+        {field:'groupName',             displayName:'Group',            width: '20%',   cellClass: 'grid-align',
+            headerTooltip: 'Group name', cellTooltip: true},
+        {field:'projectCount',          displayName:'Project',          width: '16%',   cellClass: 'grid-align',
+            headerTooltip: 'Number of projects',            aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'userCount',             displayName:'User',             width: '16%',   cellClass: 'grid-align',
+            headerTooltip: 'Number of users',               aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'defectCountTotal',      displayName:'Total',            width: '16%',   cellClass: 'grid-align',
+            headerTooltip: 'Number of all defects',         aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'defectCountFixed',      displayName:'Fixed',            width: '16%',   cellClass: 'grid-align',
+            headerTooltip: 'Number of fixed defects',       aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'defectCountDismissed',  displayName:'Dismissed',        width: '16%',   cellClass: 'grid-align',
+            headerTooltip: 'Number of dismissed defects',   aggregationType: uiGridConstants.aggregationTypes.sum}
+    ];
+
+    initialize();
+
+    function initialize() {
+        $scope.summaryGridOptions = createGrid(summaryColumnDefs);
+        removeUselessGridOptions($scope.summaryGridOptions);
```

Comment:
You don't need to follow this recommendation.
You can use addScrollbar() method instead of removeScrollbarFromGrid() method.
That's more natural.  :-)